joomla/joomla-cms
# [4.2] The "activate" tag is not needed in the user registration email template without activation

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/39115.

### Summary of Changes
The PR removes an unnecessary tag, see details https://github.com/joomla/joomla-cms/issues/39115.

### Testing Instructions
Install the build with PR, go to email templates, make sure templates `com_users.registration.user.registration_mail` and `com_users.registration.user.registration_mail_w_pw`  don't have tag `{ACTIVATE}`.

The PR is also making changes to existing sites to get rid of the tag there as well. Regular update.

### Link to documentations
Please select:
